### PR TITLE
Dynamic option for Child and Infant dropdown

### PR DESCRIPTION
**Pull request by gimperial**
The counter option for child and infant evolved based on the adult counter

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -14,4 +14,20 @@ function validateForm() {
   }
   infantsError.innerHTML = "";
   return true;
+}
+
+function dynamicdropdown(listindex)
```
Comment (tsolakoua) [inline script.js:19]: Can you rename the name of the function to follow convention naming? dynamicdropdown -> dynamicDropdown

**Comment by tsolakoua** [comment]
@gimperial amazing!! thanks so much :) I just left a minor comment!

File: script.js (hunk 1 above)
Comment (tsolakoua) [reply script.js:19]: that's also very small but maybe the listindex -> listIndex?

Comment (gimperial) [reply script.js]: My bad, I didn't notice.
All done now. Thanks

**Review by tsolakoua — APPROVED** [review]

**Pull request merged**